doc/DbStoreSizeLimits.md

```diff
@@ -0,0 +1,75 @@
+# How to configure database store size limits of Edge Hub
+
+By default, the IoT Edge system module Edge Hub stores data in an on device disk-based database store. This database store is used to store configuration data and messages.
+
+Support for an in-memory database store is also provided for customers who wish to limit the amount of disk I/O on the device and can be enabled by setting the `UsePersistentStorage` environment variable of the module to `false`.
```

Comment:
I would reword this as - Users can configure the EdgeHub to use an in-memory store instead, by setting the UsePersistentStorage environment to 'false'.